lldb/source/Utility/Stream.cpp

```diff
@@ -70,6 +72,32 @@ size_t Stream::PutCString(llvm::StringRef str) {
   return bytes_written;
 }
 
+void Stream::PutCStringColorHighlighted(llvm::StringRef text,
+                                        const char *pattern) {
+  if (!pattern) {
+    PutCString(text.data());
```

Comment:
`size_t PutCString(llvm::StringRef cstr);` so you don't need the .data();

What likely happens is that .data gives you const char* and StringRef has a constructor from const char *, so it ends up back as a string ref.

Reply:
Done! Thank you, I did not realize that.